superset-embedded-sdk/src/index.ts

```diff
@@ -38,6 +38,8 @@ export type EmbedDashboardParams = {
   mountPoint: HTMLElement
   /** A function to fetch a guest token from the Host App's backend server */
   fetchGuestToken: GuestTokenFetchFn
+  /** The dashboard UI config: hideTitle: 1; hideTab: 2; hideNav: 4; hideChartControls: 8; **/
+  uiConfig?: number
```

Comment:
We should take in individual flags here rather than making the caller figure out the bit logic

Reply:
```
uiConfig: {
    hideTitle: boolean
    hideTab: boolean
    hideNav: boolean 
    hideChartControls: boolean
}
```
something like this?